### PR TITLE
MQTT Manual config items under integration key Fan and Light platform

### PR DESCRIPTION
**Pull request by jbouwh**
## Proposed change

The way manual MQTT items are configured is changing.
The first PR will support configuring manual MQTT Lights and Fans by placing the config under the integration key.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/71676
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/fan.mqtt.markdown [hunk 1]
```diff
@@ -22,11 +22,15 @@ To enable MQTT fans in your installation, add the following to your `configurati
...
 # Example configuration.yaml entry
-fan:
-  - platform: mqtt
-    command_topic: "bedroom_fan/on/set"
+mqtt:
+  fan:
+    - command_topic: "bedroom_fan/on/set"
 ```
 
+<div class='note warning'>
```
Comment (emontnemery) [inline source/_integrations/fan.mqtt.markdown]: maybe a note is enough?
```suggestion
<div class='note'>
```

Maybe also make this linkable so we can link to it from the deprecation warning message

File: source/_integrations/fan.mqtt.markdown [hunk 2]
```diff
@@ -22,11 +22,15 @@ To enable MQTT fans in your installation, add the following to your `configurati
...
+The way manual MQTT Fans are configured has changed. Placing configurations under the `fan` platform key is deprecated.
```
Comment (emontnemery) [inline source/_integrations/fan.mqtt.markdown]: I think we should have a really short example here too to explain the difference between the old and new config.

File: source/_integrations/fan.mqtt.markdown (hunk 1 above)
Comment (jbouwh) [reply source/_integrations/fan.mqtt.markdown]: A link in the log is not clickable. The note is on top (info now)

File: source/_integrations/fan.mqtt.markdown (hunk 2 above)
Comment (jbouwh) [reply source/_integrations/fan.mqtt.markdown]: ok, but the note is place with the example configuration

File: source/_integrations/fan.mqtt.markdown (hunk 1 above)
Comment (jbouwh) [reply source/_integrations/fan.mqtt.markdown]: I added `<a id='new_format'></a>` which works. @frenck is this the right way to place a page anchor?

File: source/_integrations/fan.mqtt.markdown (hunk 2 above)
Comment (jbouwh) [reply source/_integrations/fan.mqtt.markdown]: @frenck are the page ancors ok the way I used them? `<a id='new_format'></a>`

**Comment by jbouwh** [comment]
@frenck are the page anchors okay the way I used them? <a id='new_format'></a>

**Review by frenck — APPROVED** [review]
Thanks, @jbouwh 👍

**Pull request merged**